get-toolchain.py

```diff
@@ -108,12 +108,12 @@ def main(argv):
     plat = platform()
     print("Platform:", plat)
 
-    if os.environ.get('CI', False):
-        platforms = {
-            "windows": "Windows",
-            "linux": "Linux",
-            "mac": "macOS"
-        }
+    platforms = {
```

Comment:
I think you can merge this into function `platform` in L19-L27. That is, replace the return values in L21, L23 and L25.

Reply:
Yeah, makes sense.